### PR TITLE
Adding norm operator to PyTorch

### PR DESCRIPTION
**Pull request by aksrajvanshi**
Added norm operator to PtNDArray and PyTorch.

### CODE REVIEW

File: pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_reduction.cc [hunk 1]
```diff
@@ -157,3 +157,14 @@ JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_torchCumSum(
...
   API_END_RETURN()
 }
+
+JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_torchNorm(
+    JNIEnv* env, jobject jthis, jlong jhandle, jint jord, jlongArray jaxes, jboolean jkeep_dims) {
+  API_BEGIN()
+  const auto* tensor_ptr = reinterpret_cast<torch::Tensor*>(jhandle);
+  const std::vector<int64_t> axes = djl::utils::jni::GetVecFromJLongArray(env, jaxes);
+  const auto* result_ptr =
+      new torch::Tensor(tensor_ptr->norm(c10::optional<c10::Scalar>{jord}, axes, jkeep_dims == JNI_TRUE));
```
Comment (stu1130) [inline pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_reduction.cc:167]:
```suggestion
      new torch::Tensor(tensor_ptr->norm(c10::make_optional<c10::Scalar>(jord), axes, jkeep_dims == JNI_TRUE));
```

File: pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_reduction.cc [hunk 2]
```diff
@@ -157,3 +157,14 @@ JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_torchCumSum(
...
+  return reinterpret_cast<uintptr_t>(result_ptr);
+  API_END_RETURN()
+}
```
Comment (stu1130) [inline pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_reduction.cc:170]: add new line here

File: pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_reduction.cc [hunk 3]
```diff
@@ -157,3 +157,15 @@ JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_torchCumSum(
   return reinterpret_cast<uintptr_t>(result_ptr);
   API_END_RETURN()
 }
+
+JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_torchNorm(
+    JNIEnv* env, jobject jthis, jlong jhandle, jint jord, jlongArray jaxes, jboolean jkeep_dims) {
+  API_BEGIN()
+  const auto* tensor_ptr = reinterpret_cast<torch::Tensor*>(jhandle);
+  const std::vector<int64_t> axes = djl::utils::jni::GetVecFromJLongArray(env, jaxes);
```
Comment (stu1130) [inline pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_reduction.cc:165]: you should also check empty long array

Comment (aksrajvanshi) [reply pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_reduction.cc:165]: technically, the long array can be empty when we have to give it a default value.

File: pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_reduction.cc (hunk 2 above)
Comment (stu1130) [reply pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_torch_reduction.cc:171]: did you add new line here, i am still seeing no new line in the end

**Review by stu1130 — APPROVED** [review]

**Pull request merged**